### PR TITLE
LLT-5652 Performance measurements

### PR DESCRIPTION
**Pull request by jjanowsk**
First version of performance measurement rig

Runs iperf3 via wireguard-go, main branch of NepTUN and the current
branch between two docker containers. The results are printed in the
job log. Probably we want to add some configurability to those scripts
but it is better to do it as we go, when we actually know what we need.

Most likely we want those improvements:
- host docker image and make automation for its update

### CODE REVIEW

**Review by gytsto — COMMENTED** [review]
looks great +1

**Comment by Hasan6979** [comment]
Can you add tcp tests as well ?

File: xtask/perf/left.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    private-key <(echo 0Fn5JWI1QGDiaVYLDBSLklIEBUujfpX1oH/UGI2D62k=) \
+    peer fqrU1Wk8nuyy6phf8IUDwcKK1ElslFYDyteANr2hlgM= \
+    allowed-ips 10.0.2.2/32 \
+    endpoint 176.0.0.3:51822
+ip address add dev wg2 10.0.2.1/24
+ip link set up dev wg2
+
+echo
+echo "Raw network:"
+iperf3 -i 10 -t  10 --bidir -c 176.0.0.3
```
Comment (Hasan6979) [inline xtask/perf/left.sh:35]: Add `-b 0` to really try to throttle the connection

Comment (jjanowsk) [reply xtask/perf/left.sh:35]: This is actually TCP, so `-b` is not needed. Regarding adding second protocol I think we can add it later, given that we want to merge this one rather sooner then later

Comment (Hasan6979) [reply xtask/perf/left.sh:35]: Ah. yes. right!

**Comment by Hasan6979** [comment]
+1

**Review by Hasan6979 — APPROVED** [review]

**Pull request merged**